tests/admin_docs/tests.py

```diff
@@ -220,9 +221,10 @@ class TestModelDetailView(TestDataMixin, AdminDocsTestCase):
 
     def setUp(self):
         self.client.login(username='super', password='secret')
-        self.response = self.client.get(
-            reverse('django-admindocs-models-detail',
-                    args=['admin_docs', 'person']))
+        with captured_stderr() as self.rst_stderr:
```

Comment:
rst -> docutils maybe?
I think we could put the stuff after `.get(` on the same line too if you want.
otherwise, LGTM.